cmd/minikube/cmd/config/config.go

```diff
@@ -213,6 +218,79 @@ func configurableFields() string {
 	return strings.Join(fields, "\n")
 }
 
+// AddToConfigArray adds entries to an array in the config file
+func AddToConfigArray(name string, images []string) error {
```

Comment:
Maybe a map is a better data structure here if you want to maintain a list of keys without duplication.